sdks/python/container/py39/base_image_requirements.txt

```diff
@@ -118,7 +118,7 @@ pandas==2.1.4
 parameterized==0.9.0
 pluggy==1.5.0
 proto-plus==1.24.0
-protobuf==4.25.5
+protobuf==5.28.2
```

Comment:
this is a notable change; numpy however didn't update.

Reply:
@jrmccluskey I assume this is expected.

Reply:
Don't make this jump yet, some dependencies users provide themselves like tensorflow aren't supporting protobuf 5.x yet.

Reply:
ok. Do we plan to make this jump for Beam 2.61.0? or should we pin this to 4.xx?

Reply:
@jrmccluskey are protobuf 4.x and 5.x forwards/backwards compatible?

Reply:
a smoke test would be to run a pipeline with protobuf 4.x at submission and 5.x installed in containers, and vice-versa

Reply:
As best I am aware the only incompatibility is the timestamp bound change, I can't find any documentation to the contrary

Reply:
pin protobuf to 4.25.5 for now.